apiary.apib

```diff
@@ -1164,8 +1185,12 @@ There are a lot of filtering capabilities for this report. Most of the columns r
 
 ## Campaign Lite (Campaign Lite Base)
 - `_links` (object)
-    - self
+    - `self`
         - href: https://affiliate.api.diablomedia.com/v1/campaigns/1?publisher_id=10000 (string)
+    - `suppression`
```

Comment:
Technically I think this is nullable too, if a campaign doesn't have a suppression file, this won't be present no?